configs/application/components.json

```
@@ -185,6 +185,81 @@
 					}
 				}
 			}
+		},
+		"apitest1": {
+			"window": {
+				"url": "$applicationRoot/components/apitest/test1.html",
+				"frame": false,
+				"resizable": true,
+				"autoShow": true,
+				"top": "center",
+				"left": "center",
+				"width": 800,
```

Make the component wider so the tabs fit in the initial width.
```suggestion
				"width": 925,
```